src/seq/kmer.jl

```diff
@@ -273,3 +273,21 @@ function swap{T,k}(kmer::Kmer{T,k}, i, j)
     x = ((b >> i) $ (b >> j)) & UInt64(0x03)
     return Kmer{T,k}(b $ ((x << i) | (x << j)))
 end
+
+# String literal
+# --------------
+
+macro kmer_str(seq, flag)
+    if flag == "s"
```

Comment:
There is no need to have this flag. `Kmer` is always immutable and there is no distinction of 'static' or 'dynamic'.